deploy/docker-compose.yml

```diff
@@ -40,4 +44,6 @@ services:
       - ./certbot/conf:/etc/letsencrypt
       # Mount the certbot renewal wwwroot
       - ./certbot/www:/var/www/certbot
-    entrypoint: "/bin/sh -c 'trap exit TERM; while :; do certbot renew --webroot --webroot-path /var/www/certbot/; sleep 12h & wait $${!}; done;'"
\ No newline at end of file
+    entrypoint: "/bin/sh -c 'trap exit TERM; while :; do certbot renew --webroot --webroot-path /var/www/certbot/; sleep 12h & wait $${!}; done;'"
+volumes:
```

Comment:
I am not sure what this is doing at the end of the file:
```
volumes:
   app-volume: 
```

Reply:
It's creating the shared named volume so that the `proxy` container and `osn-discovery-webui` container can share the same files. This is so that when the webui container builds the compiled files, the proxy container can see it as well and then be able to serve those compiled static files

Reply:
Okay, I don't use docker volumes very often, I usually just the local disk instead. 

Is there anything on the app-volume that would cause any of the services to stop working if it was deleted when the docker containers are not running? 
Also, if the code for osn-discovery-webui is updated, will the change made be automatically copied to the app-volume when the new container is started?

Reply:
I dont think so. The app-volume is just there so that when the web ui docker container runs, it builds the files then and there and then the proxy can use those files to serve. So if the containers aren't being run, i.e. the web site is not up, there should be no problem. The files are generated dynamically.

Reply:
Yes, any changes made to osn-discovery-webui will be automatically copied as long as the the docker compose is using the latest stable version